### DISCUSSION TITLE
Start SlingServer PC with just mains power

### DISCUSSION THREAD

**Discussion by gbjbasdw** [GerryDazoo/Slinger#243] [Q&A] [answered]
Does anyone know if you can start up a Windows PC and then start a program such as SlingServer just by applying mains power to it ?

I use Sonoff WiFi switches to power on/off my webcam and Slingbox plus HDMI to Component converter remotely when I am away from home. I would ideally like to be able to switch on/off a Windows PC running the SlingServer software too.

**Comment by GerryDazoo** [comment, marked as answer]
Most PCs have a BIOS setting  to auto power up after a power failure. You then need to make a service or setup a small batch file to run on startup. Google is your friend.

**Reply by sp222222** [reply]
OP
bios - then use this to create a service and throw the slinGER batch file into it. Works flawlessly once the win box get past POST

https://nssm.cc

enjoy
